index.js

```diff
@@ -30,6 +30,11 @@ module.exports = exports.default = function emailPropt({
   allowInvalidChars = false
 } = {}) {
   return new Promise((resolve, reject) => {
+    // some environments (e.g., cygwin) don't provide a tty
+    if (!process.stdin.setRawMode) {
```

Comment:
@mreinstein I don't intend to question your choice here, I'm just interested because I have similar problems with `process.stdin`. Do you have specific evidence for such a case?